src/libs/actions/IOU.js

```diff
@@ -72,17 +84,16 @@ function buildOnyxDataForMoneyRequest(chatReport, iouReport, transaction, chatCr
             key: `${ONYXKEYS.COLLECTION.TRANSACTION}${transaction.transactionID}`,
             value: transaction,
         },
-        ...(isNewChatReport
-            ? [
-                  {
-                      onyxMethod: Onyx.METHOD.SET,
-                      key: `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${chatReport.reportID}`,
-                      value: {
-                          [chatCreatedAction.reportActionID]: chatCreatedAction,
-                      },
-                  },
-              ]
-            : []),
+        {
+            onyxMethod: isNewChatReport ? Onyx.METHOD.SET : Onyx.METHOD.MERGE,
+            key: `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${chatReport.reportID}`,
+            value: {
+                ...(isNewChatReport ? {[chatCreatedAction.reportActionID]: chatCreatedAction} : {}),
+                [reportPreviewAction.reportActionID]: {
+                    ...(isNewReportPreviewAction ? reportPreviewAction : {created: DateUtils.getDBTime()}),
```

Comment:
Hmm, I'm not sure about this. Does it align with the deign docs? We are updating the `created` value of the action. Can we update another timestamp field?